helm/vernemq/values.yaml

```diff
@@ -15,6 +15,7 @@ fullnameOverride: ""
 
 serviceMonitor:
   create: false
+  interval: 30s
```

Comment:
is this needed here, though?

Reply:
It's needed there such that there is a default configured, and so that end users know it's a configurable value.